 #get nav status
---
int32 status